lib/util/get_min_max_distorted_amount.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const _isFinite = require('lodash/isFinite')
+const { prepareAmount } = require('bfx-api-node-util')
+
+module.exports = (args = {}, pairConfig = {}) => {
+  const { amount, sliceAmount, amountDistortion = 0 } = args
+  const { minSize, maxSize } = pairConfig
+
+  const m = amount < 0 ? -1 : 1
+
+  let maxDistortedAmount = (1 + amountDistortion) * sliceAmount
+  let minDistortedAmount = (1 - amountDistortion) * sliceAmount
```

Comment:
try this input as a test:

```js
  it('should return distorted amounts equal to the maximum order size', () => {
    const orderParams = { amount: -0.3, sliceAmount: -0.1, amountDistortion: 0.25 } // sell order
    const { minDistortedAmount, maxDistortedAmount } = getMinMaxDistortedAmount(orderParams, pairConfig)

  })
```

Reply:
best way is to go via bignumber.js instead of applying `+prepareAmount` to the end result. in this was number values are correct at all times